src/main/resources/feedbackQuestionConstSumEditFormOptionFragment.html

```diff
@@ -1,5 +1,5 @@
-<div id="constSumOptionRow-${i}-${questionNumber}">
-    <div class="input-group col-sm-12">
+<div class="margin-bottom-7px" id="constSumOptionRow-${i}-${questionNumber}">
```

Comment:
Not really. I think having such reusable and atomic classes is quite useful for the future. In fact, I could reuse some classes that were previously defined as well. I think we should have well defined increments for these classes. Then in the future, if someone wants to change to 20px (as an example), they can simply change the class.